compiler/ras/Tree.cpp

```diff
@@ -2595,8 +2595,8 @@ TR_Debug::verifyBlocksPass2(TR::Node *node)
 
       if (node->getLocalIndex() != 0)
          {
-         char buffer[100];
-         sprintf(buffer, "BLOCK VERIFICATION ERROR -- node [%s] accessed outside of its (extended) basic block: %d time(s)",
+         char buffer[150];
+         sprintf(buffer, "BLOCK VERIFICATION ERROR -- node [%s] accessed outside of its (extended) basic block: %d time(s)\n",
```

Comment:
Happy to merge this but do you have any thoughts on using `omrstr_printf` as a length checked cross-platform safe alternative?

Reply:
Seems reasonable enough. I haven't found any uses of it in the compiler, though. Is this something we intend to use more generally? Also, it looks like it requires the port library to be initialized, so it's going to depend on the progress of that work.